UnityProject/Assets/Scripts/Core/Chat/Chat.cs

```diff
@@ -505,14 +509,17 @@ public static void AddExamineMsgToClient(string message)
 	/// <param name="side">side this is being called from</param>
 	public static void AddExamineMsg(GameObject recipient, string message, NetworkSide side)
 	{
-		switch (side)
+		if (side == NetworkSide.Client)
```

Comment:
couldnt you have just put a default case instead of switching to If statement?

Reply:
yes - it was unclear to me if codacy counted the default statement when counting for its fewer-than-three-cases style. It seems like it does so I will make it a switch again